conbench/app/_plots.py

```diff
@@ -236,17 +226,28 @@ def time_series_plot(history, benchmark, run, height=250, width=1000):
     p.xaxis.major_label_orientation = 1
     p.yaxis.axis_label = axis_unit
 
-    p.line(source=source, legend_label="History", name="history")
+    hist_line = p.line(source=source, legend_label="History", name="history")
     p.line(source=source_mean, color="#ffa600", legend_label="Mean")
     p.line(source=source_alert_min, color="Silver", legend_label="+/- 5 σ")
     p.line(source=source_alert_max, color="Silver")
-    p.circle(
+    bench_circle = p.circle(
         source=source_x,
         size=8,
         color="#ff6361",
         legend_label="Benchmark",
         name="benchmark",
     )
+    p.add_tools(
+        bokeh.models.HoverTool(
+            tooltips=[
+                ("date", "$x{%F}"),
+                ("mean", "@means"),
+                ("commit", "@commits"),
+            ],
+            formatters={"$x": "datetime"},
+            renderers=[hist_line, bench_circle],
+        )
+    )
```

Comment:
Yeah, could be fun. [Looks like](https://discourse.bokeh.org/t/open-link-on-click/99/6) we'd have to delve into a bit of custom javascript to do that. There's not an easy way to do hyperlinks, especially because the hover tool always avoids the cursor

Reply:
#421